python/tvm/relay/op/nn/nn.py

```diff
@@ -202,6 +203,8 @@ def conv2d(data,
         dilation = (dilation, dilation)
     if isinstance(padding, int):
         padding = (padding, padding)
```

Comment:
Consider removing this logic since `get_pad_tuple` can also accept `int`.

Reply:
thanks, removed